### DISCUSSION TITLE
Wildcard Manager UI Suggestion

### DISCUSSION THREAD

**Discussion by cirwin518** [adieyal/sd-dynamic-prompts#127] [Ideas]
The wildcard manager is a nice addition, but I am incredibly frustrated in finding the "Delect All Wildcards" button DELETES all wildcards without a secondary safety prompt. I would love if this button was labeled properly and either hidden by default or pop ups a prompt asking if the user is sure they want to delete.

I lost days worth of custom wildcards and they did not end up in the recycling bin...

**Comment by adieyal** [comment]
Shoot - sorry to hear about that. I'll fix it.

**Reply by adieyal** [reply]
I've added a prompt confirmation - also, I'm sending the deleted files to Trash instead of explicitly deleting them.

**Reply by cirwin518** [reply]
That’s awesome! I thoroughly appreciate your work. I managed to use Chat GPT to rebuild my custom Wildcard files fairly quickly so my loss didn’t end up being too bad.

**Reply by adieyal** [reply]
Thanks for the kind words and great to hear that you could re-create your wildcards, sounds like it was a lot of work.
